src/main/connector/system/LinuxProcess/LinuxProcess.yaml

```diff
@@ -0,0 +1,53 @@
+extends:
+- ../System/System
+connector:
+  displayName: LinuxProcess
+  platforms: Any platform with LinuxOS
+  reliesOn: Linux OsCommands
+  information: Gives process specific information and metrics
+  detection:
+    connectionTypes:
+    - remote
+    - local
+    appliesTo:
+    - linux
+    criteria:
+    - type: commandLine
+      commandLine: /usr/bin/uname -o || /bin/uname -o
+      expectedResult: GNU/Linux
+      errorMessage: Not a valid Linux host.
+    tags: [ system, linux ]
+monitors:
+  process:
+    simple:
+      sources:
+        process:
+          type: commandLine
+          commandLine: ps aux | grep '${var::processNames}'
+          computes:
+          - type: awk
+            script: |
+              {
+                pid = $2
+                cpu = $3
+                mem = $4
+                command = $11
```

Comment:
`$11` will not capture the entire command. You need to concatenate from `$11` to `$NF` (with a simple `for` loop).

Reply:
The problem is that if you don't expose the entire command, it's impossible to differentiate several Java processes, as they are all called `java.exe`, which is annoying.

The problem is for processes that include varying parameters, which will increase the cardinality of the metrics for no good purpose... 🤷‍♂️